### PR TITLE
enh(mc): update meraki-restapi

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Le connecteur apporte les modèles de service suivants
 
 | Alias        | Modèle de service                                            | Description                            |
 |:-------------|:-------------------------------------------------------------|:---------------------------------------|
-| Api-Requests | Net-Cisco-Meraki-Cloudcontroller-Api-Requests-Restapi-custom | Contrôle l'utilisation de l'API Meraki |
+| Api-Requests | Net-Cisco-Meraki-Cloudcontroller-Api-Requests-Restapi-custom | Contrôle l'utilisation de l'API meraki |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:26]:
```suggestion
| Api-Requests | Net-Cisco-Meraki-Cloudcontroller-Api-Requests-Restapi-custom | Contrôle l'utilisation de l'API Meraki |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 2]
```diff
@@ -43,16 +43,17 @@ Le connecteur apporte les modèles de service suivants
...
-| Devices  | Net-Cisco-Meraki-Cloudcontroller-Devices-Restapi-custom  | Contrôle l'utilisation des équipements              | X          |
-| Networks | Net-Cisco-Meraki-Cloudcontroller-Networks-Restapi-custom | Contrôle l'utilisation des réseaux                  |            |
+| Alias       | Modèle de service                                           | Description                                         | Découverte |
+|:------------|:------------------------------------------------------------|:----------------------------------------------------|:----------:|
+| Cache       | Net-Cisco-Meraki-Cloudcontroller-Cache-Restapi-custom       | Service permettant de générer les fichiers de cache |            |
+| Devices     | Net-Cisco-Meraki-Cloudcontroller-Devices-Restapi-custom     | Contrôle l'utilisation des équipements              | X          |
+| Networks    | Net-Cisco-Meraki-Cloudcontroller-Networks-Restapi-custom    | Contrôle l'utilisation des réseaux                  |            |
+| Vpn-Tunnels | Net-Cisco-Meraki-Cloudcontroller-Vpn-Tunnels-Restapi-custom | Contrôle l'utilisation des équipements              | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:56]: des tunnels VPN?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 3]
```diff
@@ -89,56 +91,93 @@ Voici le tableau des services pour ce connecteur, détaillant les métriques rat
...
+|:----------------------------------------------------|:------|
+| *organizations*#organization.api.requests.200.count | count |
+| *organizations*#organization.api.requests.404.count | count |
+| *organizations*#organization.api.requests.429.count | count |
 
 </TabItem>
 <TabItem value="Cache" label="Cache">
 
-Ce service n'intègre pas de métriques.
+Coming soon
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:103]:
```suggestion
Ce modèle de service ne comprend pas de métriques.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 4]
```diff
@@ -89,56 +91,93 @@ Voici le tableau des services pour ce connecteur, détaillant les métriques rat
...
+| *devices*~*device_links*#link-status                           | N/A   |
+| *devices*~*device_links*#device.link.latency.milliseconds      | ms    |
+| *devices*~*device_links*#device.link.loss.percentage           | %     |
+| *devices*~*device_ports*#port-status                           | N/A   |
+| *devices*~*device_ports*#device.port.traffic.in.bitspersecond  | b/s   |
+| *devices*~*device_ports*#device.port.traffic.out.bitspersecond | b/s   |
 
 </TabItem>
-<TabItem value="Networks" label="Networks">
+<TabItem value="Network & Networks" label="Network & Networks">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:160]:
```suggestion
<TabItem value="Networks" label="Networks">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 5]
```diff
@@ -249,61 +288,61 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-| PROXYURL              | Proxy URL. E.g.: http://my.proxy:3128                                                                 |                   |             |
-| MERAKIAPIEXTRAOPTIONS | Any extra option you may want to add to every command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+| Macro                 | Description                                                                                          | Valeur par défaut | Obligatoire |
+|:----------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| MERAKIAPIHOSTNAME     | Meraki API hostname                                                                                  | api.meraki.com    | X           |
+| MERAKIAPITOKEN        | Meraki API token                                                                                     |                   | X           |
+| MERAKIAPIPROTO        | Define the protocol to reach the API                                                                 | https             |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:302]:
```suggestion
| MERAKIAPIPROTO        | Define the protocol to use to reach the API                                                                 | https             |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 6]
```diff
@@ -249,61 +288,61 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-| PROXYURL              | Proxy URL. E.g.: http://my.proxy:3128                                                                   |                   |             |
-| MERAKIAPIEXTRAOPTIONS | Any extra option you may want to add to every command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+| Macro                 | Description                                                                                          | Valeur par défaut | Obligatoire |
+|:----------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| MERAKIAPIHOSTNAME     | Meraki API hostname                                                                                  | api.meraki.com    | X           |
+| MERAKIAPITOKEN        | Meraki API token                                                                                     |                   | X           |
+| MERAKIAPIPROTO        | Define the protocol to reach the API                                                                 | https             |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:321]:
```suggestion
| MERAKIAPIPROTO        | Define the protocol to use to reach the API                                                                 | https             |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 7]
```diff
@@ -249,61 +288,61 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-| PROXYURL              | Proxy URL. E.g.: http://my.proxy:3128                                                                 |                   |             |
-| MERAKIAPIEXTRAOPTIONS | Any extra option you may want to add to every command (E.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |
+| Macro                 | Description                                                                                          | Valeur par défaut | Obligatoire |
+|:----------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| MERAKIAPIHOSTNAME     | Meraki API hostname                                                                                  | api.meraki.com    | X           |
+| MERAKIAPITOKEN        | Meraki API token                                                                                     |                   | X           |
+| MERAKIAPIPROTO        | Define the protocol to reach the API                                                                 | https             |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:341]:
```suggestion
| MERAKIAPIPROTO        | Define the protocol to use to reach the API                                                                 | https             |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 8]
```diff
@@ -472,36 +551,42 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-  --mode='devices' \
-  --hostname='api.meraki.com' \
-  --api-token='12345abcd6789efgh0123abcd4567efgh8901abcd' \
-  --proxyurl='http://proxy.mycompany:8080' \
-  --filter-device-name='centreon-par-training-ap' \
-  --critical-status='%{status} =~ /alerting/i' \
-  --critical-link-status='%{link_status} =~ /failed/i' \
-  --verbose
+	--plugin=network::cisco::meraki::cloudcontroller::restapi::plugin \
+	--mode=networks \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:560]: reprendre l'exemple précédent

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 9]
```diff
@@ -21,39 +21,40 @@ The connector brings the following service templates (sorted by the host templat
 <TabItem value="Net-Cisco-Meraki-Cloudcontroller-Restapi-custom" label="Net-Cisco-Meraki-Cloudcontroller-Restapi-custom">
 
 | Service Alias | Service Template                                             | Service Description    |
-|:------------- |:------------------------------------------------------------ |:---------------------- |
-| Api-Requests  | Net-Cisco-Meraki-Cloudcontroller-Api-Requests-Restapi-custom | Check Meraki API usage |
+|:--------------|:-------------------------------------------------------------|:-----------------------|
+| Api-Requests  | Net-Cisco-Meraki-Cloudcontroller-Api-Requests-Restapi-custom | Check meraki API usage |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:25]:
```suggestion
| Api-Requests  | Net-Cisco-Meraki-Cloudcontroller-Api-Requests-Restapi-custom | Check Meraki API usage |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 10]
```diff
@@ -21,39 +21,40 @@ The connector brings the following service templates (sorted by the host templat
...
-|:------------- |:-------------------------------------------------------- |:-------------------- |:--------- |
-| Cache         | Net-Cisco-Meraki-Cloudcontroller-Cache-Restapi-custom    | Generate cache files |           |
-| Devices       | Net-Cisco-Meraki-Cloudcontroller-Devices-Restapi-custom  | Check devices usage  | X         |
-| Networks      | Net-Cisco-Meraki-Cloudcontroller-Networks-Restapi-custom | Check networks usage |           |
+| Service Alias | Service Template                                            | Service Description  | Discovery  |
+|:--------------|:------------------------------------------------------------|:---------------------|:----------:|
+| Cache         | Net-Cisco-Meraki-Cloudcontroller-Cache-Restapi-custom       | Generate cache files |            |
+| Devices       | Net-Cisco-Meraki-Cloudcontroller-Devices-Restapi-custom     | Check devices usage  | X          |
+| Networks      | Net-Cisco-Meraki-Cloudcontroller-Networks-Restapi-custom    | Check networks usage |            |
+| Vpn-Tunnels   | Net-Cisco-Meraki-Cloudcontroller-Vpn-Tunnels-Restapi-custom | Check devices usage  | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:55]:
```suggestion
| Vpn-Tunnels   | Net-Cisco-Meraki-Cloudcontroller-Vpn-Tunnels-Restapi-custom | Check VPN tunnel usage  | X          |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 11]
```diff
@@ -88,56 +90,93 @@ Here is the list of services for this connector, detailing all metrics linked to
...
+| *devices*~*device_links*#link-status                           | N/A   |
+| *devices*~*device_links*#device.link.latency.milliseconds      | ms    |
+| *devices*~*device_links*#device.link.loss.percentage           | %     |
+| *devices*~*device_ports*#port-status                           | N/A   |
+| *devices*~*device_ports*#device.port.traffic.in.bitspersecond  | b/s   |
+| *devices*~*device_ports*#device.port.traffic.out.bitspersecond | b/s   |
 
 </TabItem>
-<TabItem value="Networks" label="Networks">
+<TabItem value="Network & Networks" label="Network & Networks">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:159]:
```suggestion
<TabItem value="Networks" label="Networks">
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 12]
```diff
@@ -249,61 +288,61 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-| PROXYURL              | Proxy URL. E.g.: http://my.proxy:3128                                                                 |                   |             |
-| MERAKIAPIEXTRAOPTIONS | Any extra option you may want to add to every command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+| Macro                 | Description                                                                                          | Default value     | Mandatory   |
+|:----------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| MERAKIAPIHOSTNAME     | Meraki API hostname                                                                                  | api.meraki.com    | X           |
+| MERAKIAPITOKEN        | Meraki API token                                                                                     |                   | X           |
+| MERAKIAPIPROTO        | Define the protocol to reach the API                                                                 | https             |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:302]:
```suggestion
| MERAKIAPIPROTO        | Define the protocol to use to reach the API                                                                 | https             |             |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 13]
```diff
@@ -249,61 +288,61 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:321]:
```suggestion
| MERAKIAPIPROTO        | Define the protocol to use to reach the API                                                                 | https             |             |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 14]
```diff
@@ -249,61 +288,61 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:341]:
```suggestion
| MERAKIAPIPROTO        | Define the protocol to use to reach the API                                                                 | https             |             |
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 15]
```diff
@@ -318,188 +357,234 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-  --mode='devices' \
-  --hostname='api.meraki.com' \
-  --api-token='12345abcd6789efgh0123abcd4567efgh8901abcd' \
-  --proxyurl='http://proxy.mycompany:8080' \
-  --filter-device-name='centreon-par-training-ap' \
-  --critical-status='%{status} =~ /alerting/i' \
-  --critical-link-status='%{link_status} =~ /failed/i' \
-  --verbose
+	--plugin=network::cisco::meraki::cloudcontroller::restapi::plugin \
+	--mode=networks \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:558]: same as FR

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md (hunk 4 above)
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:160]: Ce sont bien deux modèles de services séparés

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md (hunk 2 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:56]:
```suggestion
| Vpn-Tunnels | Net-Cisco-Meraki-Cloudcontroller-Vpn-Tunnels-Restapi-custom | Contrôle l'utilisation des tunnels VPN              | X          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 16]
```diff
@@ -472,36 +551,42 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
+	--critical-connections-assoc='' \
+	--warning-connections-dhcp='' \
+	--critical-connections-dhcp='' \
+	--warning-connections-dns='' \
+	--critical-connections-dns='' \
+	--warning-traffic-in='' \
+	--critical-traffic-in='' \
+	--warning-traffic-out='' \
+	--critical-traffic-out='' \
+	--verbose
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:583]:
```suggestion
  --plugin='network::cisco::meraki::cloudcontroller::restapi::plugin' \
  --mode='devices' \
  --hostname='api.meraki.com' \
  --api-token='12345abcd6789efgh0123abcd4567efgh8901abcd' \
  --proxyurl='http://proxy.mycompany:8080' \
  --filter-device-name='centreon-par-training-ap' \
  --critical-status='%{status} =~ /alerting/i' \
  --critical-link-status='%{link_status} =~ /failed/i' \
  --verbose
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 17]
```diff
@@ -472,36 +551,42 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-'centreon-par-training-ap#device.connections.success.count'=0;;;0; 'centreon-par-training-ap#device.connections.auth.count'=0;;;0; 
-'centreon-par-training-ap#device.connections.assoc.count'=0;;;0; 'centreon-par-training-ap#device.connections.dhcp.count'=0;;;0; 
-'centreon-par-training-ap#device.connections.dns.count'=0;;;0; 'centreon-par-training-ap#device.traffic.in.bitspersecond'=51.6626907073509b/s;;;0; 
-'centreon-par-training-ap#device.traffic.out.bitspersecond'=515.864632454924b/s;;;0;
-checking device 'centreon-par-training-ap'
-    status: online
-    connection success: 0
-    traffic in: 51.66 b/s, out: 515.86 b/s
-    link 'WAN 1' status: active
+OK: All networks are ok | '*networks*#network.connections.success.count'=12;;;0;'*networks*#network.connections.auth.count'=10;;;0;'*networks*#network.connections.assoc.count'=5;;;0;'*networks*#network.connections.dhcp.count'=;;;0;'*networks*#network.connections.dns.count'=8;;;0;'*networks*#network.traffic.in.bitspersecond'=15643b/s;;;0;'*networks*#network.traffic.out.bitspersecond'=98745b/s;;;0;
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:589]:
```suggestion
OK: Device 'centreon-par-training-ap' status: online - connection success: 0 - traffic in: 51.66 b/s, out: 515.86 b/s - link 'WAN 1' status: active | 
'devices.total.online.count'=0;;;0;1 'devices.total.offline.count'=0;;;0;1 'devices.total.alerting.count'=0;;;0;1 
'centreon-par-training-ap#device.connections.success.count'=0;;;0; 'centreon-par-training-ap#device.connections.auth.count'=0;;;0; 
'centreon-par-training-ap#device.connections.assoc.count'=0;;;0; 'centreon-par-training-ap#device.connections.dhcp.count'=0;;;0; 
'centreon-par-training-ap#device.connections.dns.count'=0;;;0; 'centreon-par-training-ap#device.traffic.in.bitspersecond'=51.6626907073509b/s;;;0; 
'centreon-par-training-ap#device.traffic.out.bitspersecond'=515.864632454924b/s;;;0;
checking device 'centreon-par-training-ap'
    status: online
    connection success: 0
    traffic in: 51.66 b/s, out: 515.86 b/s
    link 'WAN 1' status: active
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 18]
```diff
@@ -318,188 +357,234 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
+	--critical-connections-assoc='' \
+	--warning-connections-dhcp='' \
+	--critical-connections-dhcp='' \
+	--warning-connections-dns='' \
+	--critical-connections-dns='' \
+	--warning-traffic-in='' \
+	--critical-traffic-in='' \
+	--warning-traffic-out='' \
+	--critical-traffic-out='' \
+	--verbose
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:581]:
```suggestion
  --plugin='network::cisco::meraki::cloudcontroller::restapi::plugin' \
  --mode='devices' \
  --hostname='api.meraki.com' \
  --api-token='12345abcd6789efgh0123abcd4567efgh8901abcd' \
  --proxyurl='http://proxy.mycompany:8080' \
  --filter-device-name='centreon-par-training-ap' \
  --critical-status='%{status} =~ /alerting/i' \
  --critical-link-status='%{link_status} =~ /failed/i' \
  --verbose
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 19]
```diff
@@ -318,188 +357,234 @@ yum install centreon-plugin-Network-Cisco-Meraki-Restapi
...
-'centreon-par-training-ap#device.connections.success.count'=0;;;0; 'centreon-par-training-ap#device.connections.auth.count'=0;;;0; 
-'centreon-par-training-ap#device.connections.assoc.count'=0;;;0; 'centreon-par-training-ap#device.connections.dhcp.count'=0;;;0; 
-'centreon-par-training-ap#device.connections.dns.count'=0;;;0; 'centreon-par-training-ap#device.traffic.in.bitspersecond'=51.6626907073509b/s;;;0; 
-'centreon-par-training-ap#device.traffic.out.bitspersecond'=515.864632454924b/s;;;0;
-checking device 'centreon-par-training-ap'
-    status: online
-    connection success: 0
-    traffic in: 51.66 b/s, out: 515.86 b/s
-    link 'WAN 1' status: active
+OK: All networks are ok | '*networks*#network.connections.success.count'=12;;;0;'*networks*#network.connections.auth.count'=10;;;0;'*networks*#network.connections.assoc.count'=5;;;0;'*networks*#network.connections.dhcp.count'=;;;0;'*networks*#network.connections.dns.count'=8;;;0;'*networks*#network.traffic.in.bitspersecond'=15643b/s;;;0;'*networks*#network.traffic.out.bitspersecond'=98745b/s;;;0;
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:587]:
```suggestion
OK: Device 'centreon-par-training-ap' status: online - connection success: 0 - traffic in: 51.66 b/s, out: 515.86 b/s - link 'WAN 1' status: active | 
'devices.total.online.count'=0;;;0;1 'devices.total.offline.count'=0;;;0;1 'devices.total.alerting.count'=0;;;0;1 
'centreon-par-training-ap#device.connections.success.count'=0;;;0; 'centreon-par-training-ap#device.connections.auth.count'=0;;;0; 
'centreon-par-training-ap#device.connections.assoc.count'=0;;;0; 'centreon-par-training-ap#device.connections.dhcp.count'=0;;;0; 
'centreon-par-training-ap#device.connections.dns.count'=0;;;0; 'centreon-par-training-ap#device.traffic.in.bitspersecond'=51.6626907073509b/s;;;0; 
'centreon-par-training-ap#device.traffic.out.bitspersecond'=515.864632454924b/s;;;0;
checking device 'centreon-par-training-ap'
    status: online
    connection success: 0
    traffic in: 51.66 b/s, out: 515.86 b/s
    link 'WAN 1' status: active
```

File: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md (hunk 11 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:159]:
```suggestion
<TabItem value="Network, Networks" label="Network, Networks">
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md [hunk 20]
```diff
@@ -89,56 +91,93 @@ Voici le tableau des services pour ce connecteur, détaillant les métriques rat
...
+| *devices*~*device_links*#link-status                           | N/A   |
+| *devices*~*device_links*#device.link.latency.milliseconds      | ms    |
+| *devices*~*device_links*#device.link.loss.percentage           | %     |
+| *devices*~*device_ports*#port-status                           | N/A   |
+| *devices*~*device_ports*#device.port.traffic.in.bitspersecond  | b/s   |
+| *devices*~*device_ports*#device.port.traffic.out.bitspersecond | b/s   |
 
 </TabItem>
-<TabItem value="Networks" label="Networks">
+<TabItem value="Network & Networks" label="Network & Networks">
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md:160]:
```suggestion
<TabItem value="Network, Networks" label="Network, Networks">
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**